js/ui/views/search.es6.js

```diff
@@ -32,8 +32,10 @@ Search.prototype = $.extend({},
         },
 
         _handleSubmit: function (e) {
-            console.log(`Search submit for ${this.$input.val()}`)
-            this.model.doSearch(this.$input.val());
+            let searchVal = encodeURIComponent(this.$input.val())
```

Comment:
shouldn't this be done in the model

Reply:
Agreed, this should be done in the model. Other than that, this is fine.